commons/src/main/java/org/eclipse/kapua/commons/util/log/ConfigurationPrinter.java

```diff
@@ -19,6 +19,8 @@
 
```

Comment:
This class has been generalized a bit to improve its reusability from different contexts